### ISSUE TITLE
NBE Mobile

### ISSUE THREAD

**Issue by aly-k** [PrivSec-dev/banking-apps-compat-report#459]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### App name

NBE Mobile

### Link to app

https://play.google.com/store/apps/details?id=com.ofss.obdx.and.nbe.com.eg&hl=en

### App version

v1.0.33

### Country of the app

Egypt

### Build Number

2024060500

### Device list

Pixel 7

### Profile app tested in

Work profile (Add details in extra notes below, i.e., device manager app name)

### Google Play installed?

Installed

### Where did you install this app from?

Google Play Store

### Google Play services Network permission revoked?

- [ ] Revoked
- [X] Not revoked
- [ ] I did not have Google Play services installed

### Native code debugging

- [X] Allowed
- [ ] Blocked

### Exploit protection compatibility mode

- [X] Enabled
- [ ] Disabled

### Memory tagging extension (MTE)

- [ ] Enabled
- [ ] Disabled
- [X] N/A

### Stock OS compatibility

- [ ] Works
- [ ] Does not work
- [X] Not tested

### NFC payments

- [ ] Works
- [ ] Works but requires another service
- [ ] Does not work
- [ ] N/A (Not supported by app)
- [X] Not tested

### Description of the app's functionality

Denied usage at launch with the error message:
> Rooted device is not supported

Please note that my device is NOT rooted. I am using Graphene OS as is.

### Are there any extra notes you think users should know about?

Work profile is managed by shelter app

**Comment by spring-onion** [comment]
Yeah the "your device is rooted" error apps throw at you is just a catch all for everything they don't like, failing play integrity is one of those things. Please send them the [attestation compatibility guide](https://grapheneos.org/articles/attestation-compatibility-guide) so they can start supporting GrapheneOS.

**Comment by aly-k** [comment]
@spring-onion Would you be interested in basic comparability reports for all financial apps in Egypt ? The reports would only test which apps implement safety net/play integrity API restrictions, not the full functionality of the app.
I am moving away from GOS due to bugs that make it non viable for daily usage so a prompt reply would be appreciated.

**Comment by spring-onion** [comment]
@aly-k Please retest with release [2025012700](https://grapheneos.org/releases#2025012700).

**Comment by aly-k** [comment]
Retested. Checks play integrity and crashes on launch.
